packages/next/src/server/lib/patch-fetch.ts

```diff
@@ -727,6 +728,15 @@ export function createPatchedFetcher(
                     )
                     .finally(handleUnlock)
 
+                  const waitUntil = getBuiltinRequestContext()?.waitUntil
```

Comment:
Instead of pulling in the `RequestContext` here we could push this onto the `pendingRevalidates` array which already gets wired to the available `waitUntil`.

Reply:
I see, yes thank you very much, being unfamiliar with the codebase I didn't notice this `pendingRevalidates` logic, I'll make the change right away 🙏